server.go

```diff
@@ -108,5 +109,17 @@ func (s *Server) GetPressedKeys(args *NotImpl, ret *GetPressedKeysRetval) error
 	return nil
 }
 
+func (s *Server) ReadTouchPoint(args *NotImpl, ret *touch.Point) error {
+
+	s.Device.mu.Lock()
+	*ret = s.Device.TouchPoint
+	s.Device.TouchPoint.X = 0
+	s.Device.TouchPoint.Y = 0
+	s.Device.TouchPoint.Z = 0
```

Comment:
I don't think I'm needed here.
I think TouchPoint should keep the same value if the mouse pointer stays in the same place during drag.

Why is it cleared to 0 here?

Reply:
You're right. I tested with a very low polling frequency and didn't notice this. I keep track of the drag in progress now.